yarn.lock

```diff
@@ -1921,16 +1921,6 @@ set-blocking@^2.0.0:
   resolved "https://registry.yarnpkg.com/set-blocking/-/set-blocking-2.0.0.tgz#045f9782d011ae9a6803ddd382b24392b3d890f7"
   integrity sha1-BF+XgtARrppoA93TgrJDkrPYkPc=
 
-set-value@^2.0.0, set-value@^2.0.1:
```

Comment:
Huh. Why was this removed?

Reply:
Ah, it was out of date on `master`. Addressed in #60

Reply:
I think I erroneously included them during rebasing/merging. Running `yarn why set-value` on `master` yields no results.

Reply:
Fixed by #60